solidity/utils/test_deployment.js

```diff
@@ -249,6 +260,7 @@ const run = async () => {
     await execute(converterFactory.methods.registerTypedConverterFactory(liquidTokenConverterFactory._address));
     await execute(converterFactory.methods.registerTypedConverterFactory(liquidityPoolV1ConverterFactory._address));
     await execute(converterFactory.methods.registerTypedConverterFactory(standardPoolConverterFactory._address));
+    await execute(converterFactory.methods.registerTypedConverterFactory(fixedRatePoolConverterFactory._address));
```

Comment:
no need to add the fixed rate converter to the test deployment

Reply:
Note that these two contract (the converter and its factory) will not be verified on etherscan.

Reply:
sure

Reply:
sure

Reply:
?

Reply:
Fixed.